Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DartProblemsViewPanel.java

```diff
@@ -14,6 +19,13 @@ public DartProblemsViewPanel(Project project) {
   @Override
   protected void fillRightToolbarGroup(DefaultActionGroup group) {
     super.fillRightToolbarGroup(group);
+    group.add(new AnAction(DartBundle.message("dart.reanalyze.action.name"), DartBundle.message("dart.reanalyze.action.description"),
```

Comment:
'ToolWindowManager.getInstance(project).getToolWindow(DartProblemsViewImpl.TOOLWINDOW_ID) != null'
looks like a valid and simple implementation of isEnabled()